Sources/ArcGISToolkit/Common/NavigationLayer/NavigationLayer.swift

```diff
@@ -77,7 +77,7 @@ struct NavigationLayer<Content: View>: View {
                     root(model)
                         .transition(model.transition)
                 } else if let presented = model.presented?.view {
-                    AnyView(presented())
+                    AnyView(presented)
```

Comment:
Thank you for making this change.

It might be good to move this and the associated changes in `NavigationLayerModel` into a dedicated PR.